setup.py

```diff
@@ -216,12 +218,14 @@ def members_appended(*ds):
 
     if cythonize and cythonizing:
         cython_opts = dict(
-            # compile .pyx extensions to .c in parallel
-            nthreads=cpu_threads + 1,
             # default language_level will be '3str' starting from Cython 3.0.0,
             # but old cython versions (< 0.29) do not know that, thus we use 3 for now.
             compiler_directives={'language_level': 3},
         )
+        if os.name != 'nt':
+            # compile .pyx extensions to .c in parallel, does not work on windows
+            cython_opts['nthreads'] = cpu_threads + 1
```

Comment:
would it work on windows if the nthreads value would be just 1?

Reply:
Sadly not. But I have to investigate why. Somhow the compilation ends up in an endless cycle... So for the moment I just decided to disable parallel compilation.

Reply:
It tries to fork something (fork is not a thing on windows) fork fails and instead launches completly new instance that then tries to fork.